pkg/hyper/hyper.go

```diff
@@ -68,7 +70,16 @@ func (h *Runtime) StopPodSandbox(podSandBoxID string) error {
 // DeletePodSandbox deletes the sandbox. If there are any running containers in the
 // sandbox, they should be force deleted.
 func (h *Runtime) DeletePodSandbox(podSandBoxID string) error {
-	return fmt.Errorf("Not implemented")
+	request := types.PodRemoveRequest{
+		PodID: podSandBoxID,
+	}
+
+	_, err := h.client.client.PodRemove(context.Background(), &request)
```

Comment:
@xlgao-zju Thanks for the PR. Need to process pod already nonexistent scenario here. If the sandbox has already been nonexistent, `DeletePodSandbox` should return nil.